admin/app/controllers/authenticated/certifications/scoring-simulation.js

```diff
@@ -0,0 +1,67 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+export default class ConfigurationController extends Controller {
+  @tracked score = null;
+  @tracked capacity = null;
+  @tracked simulatorReport = null;
+  @tracked errors = [];
+  @service store;
+
+  ERRORS = {
+    SCORE: 'Merci d’indiquer un score en Pix compris entre O et 896',
+    BOTH_INPUT_FILLED: 'Merci de ne renseigner que l’un des champs “Score global en Pix” ou “Capacité”',
+    BOTH_INPUT_EMPTY: "Merci de renseigner au moins l'un des deux champs",
+  };
+
+  @action
+  async onGenerateSimulationProfile(event) {
+    event.preventDefault();
+    this._cleanErrors();
+    this.checkFormValidity();
+    const adapter = this.store.adapterFor('scoring-and-capacity-simulator-report');
+    const isFormInvalid = (!this.score && !this.capacity) || this.errors.length > 0;
```

Comment:
Il y a plusieurs moyen de savoir s'il y a des erreurs. Mais reprendre les tests deja effectué dans checkFormValidity semble etre redondant (en fait on reprend uniquement le 1er test de checkFormValidity)
On peut par exemple renvoyer true|false sur `checkFormValidity`ou encore verifier la taille de this.errors
`const isFormInvalid = this.checkFormValidity;`